Integration/Scripted REST Api/DomainSeperation/create.js

```diff
@@ -0,0 +1,98 @@
+/**
+ * -----------------------------------------------------------------------------
+ * Hacktoberfest Contribution - ServiceNow Scripted REST API

```

Comment:
please remove references to hacktoberfest, it is not relevant to the code snippet itself